### PR TITLE
Timeout SimpleFIN sync calls after 60 seconds

### PR DESCRIPTION
**Pull request by psybers**
Currently, if something goes wrong with the bank sync, the client essentially just 'hangs' while trying to sync and shows the account(s) as trying to sync forever.  They do not seem to time out.

This places a 60-second limit on the call, and if it times out it then proceeds to indicate the link might be broken and users should relink the account.

### CODE REVIEW

**Comment by psybers** [comment]
Related https://github.com/actualbudget/actual-server/pull/375

**Review by youngcw — DISMISSED** [review]

File: packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx [hunk 1]
```diff
@@ -35,6 +35,12 @@ function getErrorMessage(type, code) {
     case 'RATE_LIMIT_EXCEEDED':
       return 'Rate limit exceeded for this item. Please try again later.';
 
+    case 'INVALID_ACCESS_TOKEN':
+      return 'Your SimpleFIN Access Token is no longer valid.  Please reset and generate a new token.';
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx:39]: **🥜 nitpick:** ‏
```suggestion
      return 'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.';
```

File: packages/desktop-client/src/components/modals/CreateAccountModal.tsx [hunk 2]
```diff
@@ -62,36 +62,46 @@ export function CreateAccountModal({
...
-    });
+        newAccounts.push(newAccount);
+      }
+
+      actions.pushModal('select-linked-accounts', {
+        accounts: newAccounts,
+        syncSource: 'simpleFin',
+      });
+    } catch (err) {
+      console.error(err);
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/modals/CreateAccountModal.tsx:100]: **👏 praise:** ‏nice

**Review by MatissJanis — APPROVED** [review]
Thanks for adding this! Just one small nitpick. LMK what you think (we can proceed without the change).

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**